medusa/server/web/home/add_shows.py

```diff
@@ -308,7 +310,7 @@ def popularShows(self):
 
         try:
             recommended_shows = ImdbPopular().fetch_popular_shows()
-        except (StandardError, requests.RequestException) as e:
+        except (RequestException, socket.gaierror, StandardError) as e:
```

Comment:
@p0psicles because of this errors: https://github.com/pymedusa/Medusa/issues/1980